portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,42 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Jennifer Dai's Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <link href="https://fonts.googleapis.com/css2?family=PT+Sans+Narrow:wght@400;700&display=swap" rel="stylesheet">
   </head>
   <body>
+      <script src="script.js"></script>
```

Comment:
I meant to remove this line. I moved the script tag to the end of the body because I am currently working on changing the image display to a slideshow and the simplest way I could find to do it requires that the script be at the end to display the images.